.github/workflows/ci.yml

```diff
@@ -406,6 +413,11 @@ jobs:
         cd build
         cmake -DROBOTOLOGY_ENABLE_DYNAMICS_FULL_DEPS:BOOL=OFF .
 
+    - name: Install dependencies unsupported in Ubuntu 18.04
+      if: contains(matrix.os, '20.04')
```

Comment:
Why the condition is different here?

Reply:
Because it is for installing dependencies that are not available on 18.04, so it is run on Ubuntu 20.04 .

Reply:
Sorry, I should have been more precise. I meant with respect to the similar if clause above at  line 231

Reply:
I saw later that one applies to docker images and this to os. Not important at all

Reply:
> 
> 
> I saw later that one applies to docker images and this to os. Not important at all

Yes, that's it. The clause are different as for the docker images you need to check the used docker image, not the base os.